src/main/cliapp/src/components/datatables/DiseaseAnnotationsComponent.js

```diff
@@ -43,7 +44,10 @@ export const DiseaseAnnotationsComponent = () => {
     const toast_topleft = useRef(null);
     const toast_topright = useRef(null);
 
-
+    const negatedTerms = [
+        { name: 'True' },
+        { name: 'False' }
+    ];
```

Comment:
These terms should come from the `ControlledVocabularyService` just setup a list in there and give it a name.

Reply:
exactly 
exactly like this one - useQuery(['diseaseRelationTerms'],
        () => controlledVocabularyService.getTerms('disease_relation_terms'), {
            onSuccess: (data) => {
                setDiseaseRelationTerms(data)
            }
        }
    )

Reply:
yes but the name is: `generic_boolean_terms`

Reply:
yea..got it...thanks